cli/flox/src/commands/init/node.rs

```diff
@@ -622,72 +916,19 @@ impl Node {
         self.single_option_prompt()
     }
 
-    /// Prompt whether to install npm or yarn when either is viable
-    async fn prompt_for_package_manager(
-        &mut self,
-        flox: &Flox,
-        yarn_install: YarnInstall,
-        node_install: NodeInstall,
-    ) -> Result<bool> {
-        let yarn_version = Self::format_version_or_empty(yarn_install.yarn.version.as_ref());
-        let yarn_node_version = Self::format_version_or_empty(yarn_install.node.version.as_ref());
-        let node_version = Self::format_version_or_empty(
-            match &node_install.node {
-                Some(found_node) => found_node.clone(),
-                None => find_compatible_package(flox, "nodejs", None).await?,
-            }
-            .version
-            .as_ref(),
-        );
+    /// Prompt whether to install yarn
+    fn prompt_with_yarn(&self, yarn_install: &YarnInstall) -> Result<bool> {
```

Comment:
I renamed the old function above, so this one is only used in the "only install yarn" case.